helm/CHANGELOG.md

```diff
@@ -1,9 +1,15 @@
 ## v2.0.0
 
-**Breaking changes:**
+### Breaking changes
 
-- Flattened some keys of the helm chart
+- Helm Chart values flattened when possible (#393, @mauriciopoppe)
   - Flattened top level `common` dictionary, all the keys are now at the top level.
   - Flattened top level `daemonset` dictionary, all the keys are now at the top level.
 - `rbac.pspEnabled` removed
 - `useAlphaAPI` removed
+
+### Features
```

Comment:
The helm chart release might not be tied to a git tag, I think it's released by modifying the Chart.yaml file which through a post submit job will trigger a helm chart release.

https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/pull/386 might be part of 2.0.0 too because we want to remove a few more properties in the chart, anyways, I'm going to update this file a little bit more before doing the helm chart release.